settings.py

```diff
@@ -126,3 +133,12 @@
 #     "password ": "chaos",
 #     "db ": "db"
 # }
+
+SOURCE_PARTS_NAME = {
+    "system": ["ansible/*", "etc/*", "memoize/*", "puppet/*", "startup.d/*", "chaos_wrapper.sh"],
+    "core": ["cron/*", "chaos.py"],
```

Comment:
Maybe separate out files in `cron`, as they handle very different parts of chaos.